lib/rules/negation.ts

```diff
@@ -53,13 +63,12 @@ export default createRule("negation", {
                         // All other character sets are either case-invariant
                         // (/./, /\s/, /\d/) or inconsistent (/\w/).
 
-                        // FIXME: TS Error
-                        // @ts-expect-error -- FIXME
-                        const ccSet = toCharSet(ccNode, flags)
+                        // since we know that the property doesn't contain strings,
+                        // we can just get the characters of the character class
+                        // without worrying about the strings
+                        const ccSet = toUnicodeSet(ccNode, flags).chars
```

Comment:
I would like to get rid of the `.chars` here. The reasoning explained in the comment isn't trivial, so I would like to simplify it to the code I suggested.

Unfortunately, we can't compare a `UnicodeSet` to a `CharSet` right now, but I'm currently working on that. I'm currently working on a patch for refa and RAA which will include some fixes + more ergonomics for `UnicodeSet`s. So my suggested code will work then.

```suggestion
                        const ccSet = toUnicodeSet(ccNode, flags)
```